integration/templates/ingress_controller_basic_values.go

```diff
@@ -17,7 +17,7 @@ controller:
 
   image:
     repository: giantswarm/nginx-ingress-controller
-    tag: 0.23.0
```

Comment:
it was not `0.24.1` before, make senses to change atm?

Reply:
It is only informational, but I would like to have it aligned. @rossf7 WDYT?

Reply:
Yes this is just for the e2e test and I don't like that we need it. But yes it should be aligned I guess this just got missed last time.